srv/salt/ceph/stage/discovery/default.sls

```diff
@@ -4,9 +4,17 @@ ready:
     - name: minions.ready
     - timeout: {{ salt['pillar.get']('ready_timeout', 300) }}
 
+refresh_pillar0:
+  salt.state:
+    - tgt: {{ salt['pillar.get']('master_minion') }}
```

Comment:
The refresh is for the Stage 1 runner since it can generate roles (e.g. custom rgw and ganesha roles) from the pillar data.  Stage 2 will refresh all minions.